pkg/limatmpl/locator.go

```diff
@@ -285,3 +295,55 @@ func InstNameFromYAMLPath(yamlPath string) (string, error) {
 	}
 	return s, nil
 }
+
+func TransformCustomURL(ctx context.Context, locator string) (string, error) {
+	u, err := url.Parse(locator)
+	if err != nil || len(u.Scheme) <= 1 {
+		return locator, nil
+	}
+
+	if u.Scheme == "template" {
+		if u.Opaque != "" {
+			return locator, nil
+		}
+		// Fix malformed "template:" URLs.
+		newLocator := "template:" + path.Join(u.Host, u.Path)
+		logrus.Warnf("Template locator %q should be written %q", locator, newLocator)
```

Comment:
```suggestion
		logrus.Warnf("Template locator %q should be written %q since Lima v2.0", locator, newLocator)
```

Reply:
This needs to be updated too
https://github.com/lima-vm/lima/blob/94ba0e1c742bad6cd76452b07f2161bc9d720971/cmd/limactl/completion.go#L27-L49

Reply:
I updated `cmd/`, `hack/`, and `README.md`.

I've not yet updated `templates/` and `website/`. I assume we only want to update these close to the release, or should I update everything right away?

Reply:
> I've not yet updated `templates/`

And I had to revert `hack/test-upgrades.sh` to keep the command compatible with the old Lima version.